package.json

```diff
@@ -1,44 +1,75 @@
 {
-  "name": "webpack-react-practice",
+  "name": "FuBoFleaMarket",
   "version": "1.0.0",
-  "description": "First react project",
+  "description": "A flea market web site for fu-bo",
   "main": "app/app.js",
   "scripts": {
-    "test": "test",
     "build": "webpack",
+    "eslint": "eslint --fix --ext .js app test",
     "start": "webpack-dev-server --devtool eval --progress --colors --hot --content-base build",
-    "eslint": "eslint --ext .js src"
+    "test": "cross-env NODE_ENV=test mocha --recursive --compilers js:babel-register --require ./test/setup.js --reporter mochawesome",
+    "test:watch": "npm test -- --watch"
   },
-  "author": "",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Fu-Bo-Tech/fubo-flea-market.git"
+  },
+  "author": "ben196888, ElaineHuang",
   "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Fu-Bo-Tech/fubo-flea-market/issues"
+  },
+  "homepage": "https://github.com/Fu-Bo-Tech/fubo-flea-market#readme",
   "devDependencies": {
-    "autoprefixer-loader": "^3.2.0",
     "babel-core": "^6.10.4",
     "babel-eslint": "^6.1.2",
     "babel-loader": "^6.2.4",
+    "babel-plugin-transform-object-rest-spread": "^6.8.0",
+    "babel-polyfill": "^6.9.1",
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-react": "^6.11.1",
+    "babel-register": "^6.11.5",
```

Comment:
All subsequent files required by node with the extensions .es6, .es, .jsx and .js will be transformed by Babel.